fdm-app/app/components/custom/atlas/atlas-sources.tsx

```
@@ -1,14 +1,113 @@
+}: {
+    id: string
+    availableLayerId: string
+    fieldsData: FeatureCollection
+    setFieldsData: React.Dispatch<
+        React.SetStateAction<FeatureCollection>
+    > | null
+    children: JSX.Element
+}) {
+    const { current: map } = useMap()
+
+    useEffect(() => {
```

Biome (1.9.4)

[error] This hook specifies more dependencies than necessary: fieldsData

Outer scope values aren't valid dependencies because mutating them doesn't re-render the component.


(lint/correctness/useExhaustiveDependencies)